composer.json

```diff
@@ -122,7 +122,9 @@
         "phpunit/phpunit": "4.5.0",
 
         "mmoreram/php-formatter": "1.0.4",
-        "fabpot/php-cs-fixer": "1.4.2"
+        "fabpot/php-cs-fixer": "1.4.2",
+        "visithor/visithor-bundle": "0.1.*@dev",
```

Comment:
I you use the ~ operator, then you will be able to go until `1.4.3` and this is something I really want to avoid, as each version contains new rules, and make our code different. This version will be changed by hand in each bamboo/elcodi version.